.github/workflows/cd-workflow.yml

```
@@ -0,0 +1,137 @@
+              fi
+            }
+
+            choose_container
+
+            echo "🟢 NEW_CONTAINER: $NEW_CONTAINER"
+            echo "⚫️ OLD_CONTAINER: $OLD_CONTAINER"
+            echo "📦 NEW_PORT: $NEW_PORT"
+            
+            echo "🐳 Pulling latest Docker image from DockerHub"
+            docker pull ${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_IMAGE }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)